### PR TITLE
Upgrading to Entities 0.0.12-preview.8

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Upgrading to latest entities package

#### Tests
Ran the tests in Unity

#### Documentation
don't think this needs to be documented

### CODE REVIEW

File: workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 using Generated.Improbable;
 using Generated.Improbable.Transform;
 using Improbable.Gdk.Core;
+using Unity.Collections;
```
Comment (jared-improbable) [inline workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs]: Intentional?

Comment (jessicafalk) [reply workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs]: nope, will remove :)

Comment (zeroZshadow) [reply workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs]: IF that was not intentional, why didn't the lint pick it up?

Comment (jessicafalk) [reply workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs]: good question! I didn't lint locally, but the TC lint also didn't pick it up

**Pull request merged**